src/lib/fetcher/fetcher.ts

```diff
@@ -77,10 +89,12 @@ export default class Fetcher<T> {
       .filter(i => results[i] instanceof Error);
 
     results.map(result => {
-      this.logger.info(
-        'Results Data:',
-        JSON.stringify((result as any).data, null, 4).substring(0, 500),
-      );
+      if(!(result instanceof Error)) {
+        this.logger.info(
+          'Results Data:',
+          JSON.stringify((result as any).data, null, 4).substring(0, 500),
+        );
+      }
```

Comment:
Not sure I understand the reasoning behind this change. Can you elaborate, please?

Reply:
@Verisana I moved fix from @skypper 's PR https://github.com/paraswap/paraswap-dex-lib/pull/353/files#diff-c1f4c6763e1a3fc7055ef43348ac06e8bc71935bf34a82cc616e5d976ec4c8d6R83

Reply:
Thanks, got it